source/Applications/Advanced/HandEyeCalibration/PoseConversions/PoseConversions.cpp

```diff
@@ -215,11 +221,11 @@ Eigen::Array3d rotationMatrixToRollPitchYaw(const Eigen::Matrix3d &rotationMatri
 {
     switch(rotation)
     {
-        case RotationConvention::XYZ_Intrinsic: return rotationMatrix.eulerAngles(0, 1, 2);
-        case RotationConvention::XYZ_Extrinsic: return rotationMatrix.eulerAngles(2, 1, 0).reverse();
-        case RotationConvention::ZYX_Intrinsic: return rotationMatrix.eulerAngles(2, 1, 0).reverse();
-        case RotationConvention::ZYX_Extrinsic: return rotationMatrix.eulerAngles(0, 1, 2);
-        case RotationConvention::NOF_ROT: break;
+        case RotationConvention::zyxExtrinsic:
+        case RotationConvention::xyzIntrinsic: return rotationMatrix.eulerAngles(0, 1, 2);
+        case RotationConvention::xyzExtrinsic:
+        case RotationConvention::zyxIntrinsic: return rotationMatrix.eulerAngles(2, 1, 0).reverse();
+        case RotationConvention::nofROT: break;
```

Comment:
Did you notice that code can be shrinked or clang-tidy?

Reply:
I was aware of the redundant branches, but thought at the time it read better. clang-tidy didn't agree. I didn't want to disable the rule.

Reply:
I am just impressed for all it can detect..